DiscordBot/Commands/TicTacToeCommand.cs

```diff
@@ -0,0 +1,191 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus.Entities;
+using Microsoft.CSharp.RuntimeBinder;
+using TicTacToe;
+using static TicTacToe.SquareState;
+using static TicTacToe.GameState;
+
+namespace DiscordBot.Commands
+{
+    public class TicTacToeCommand : ICommand
+    {
+        private TicTacToeManager Manager { get; set; }
+
+        private Dictionary<char, SquareState> translationDict = new Dictionary<char, SquareState>
+        {
+            {'❌', X},
+            {'⭕', O},
+            {'⬜', Empty},
+        };
+
+        private string InvalidText = "Invalid screen";
+        private string WinText = "You won!!";
+        private string LoseText = "You lost :(";
+        private string DrawText = "Game drawn";
+
+        public TicTacToeCommand(TicTacToeManager manager)
+        {
+            Manager = manager;
+        }
+
+        public async Task Run(DiscordMessage msg)
+        {
+            var splitMsg = CommandParser.Split(msg.Content);
+
+            switch (splitMsg[1])
```

Comment:
Really like the usage of methods `StartText`, `StopText`...
But make it so the methods send the text themselves, not return it, and rename them to `StartSubCommand`, `StopSubCommand`...
Essentially make them 'handlers' for the commands

Reply:
u hate me cause you ain't me :D 🥇